maximeish/MyBrand-Maxime
# login page for blog

#### What does this PR do?
Gives details about the login page feature to be merged into develop

#### Description of Task to be completed?
Add blog login page to the brand website

#### How should this be manually tested?
Check if all the necessary HTML and CSS files are present, the page should fill the whole page with no scrollbar

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate
![login](https://user-images.githubusercontent.com/54126307/87799150-2eb6f280-c84d-11ea-9ba4-d284f16313f9.png)


#### Questions:
